i18n/zh/docusaurus-plugin-content-docs/current/about/_category_.yml

```diff
@@ -0,0 +1,7 @@
+position: 1 #float position is supported
```

Comment:
As a user of this language, it's frustrating to land on a page like this that makes it seem like a translation exists, but it actually doesn't. Better to not publish.